src/when/definitions.js

```diff
@@ -83,6 +84,9 @@ export const stepDefinitions = [
 			}
 		})
 		.withDocs("Use TopHat in your application to log into Nice accounts. Username and Password should be names of environment variables "),
+
+	When(/^I focus on the element "([^"]*)?"$/, focusElement).withDocs(
```

Comment:
Please remove the question mark that makes the capturing group optional, we require a selector here
```suggestion
	When(/^I focus on the element "([^"]+)"$/, focusElement).withDocs(
```